iOS/Bugsnag.xcodeproj/project.pbxproj

```diff
@@ -1514,6 +1518,7 @@
 				);
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = YES;
```

Comment:
Is this change required?

Reply:
Yes.  This change allows the CI system (which runs MacOS 10.14 - without Catalyst support) to (trivially) find and replace the line with `SUPPORTS_MACCATALYST = NO;` and allow the static library to build under CI conditions.  A manual test of a Catalyst-compatible static library build on 10.15 is also performed during UAT.